src/pages/api/urls/save/[user_id].ts

```diff
@@ -34,6 +34,7 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
         {
           real_url: url,
           slug: sanitizeSlug(slug),
+          is_dynamic: ~~is_dynamic,
```

Comment:
How if we simplify the code? `~~` seems not familiar enough for most of developers.

```suggestion
          is_dynamic: is_dynamic ? 1 : 0,
```

Reply:
That's are simplified already IMO, but yeah it's not obvious compared to ternary condition. I'll change it.